hypervisor/src/mshv/mod.rs

```diff
@@ -3,17 +3,16 @@
 // Copyright © 2020, Microsoft Corporation
 //
 
-use crate::arch::emulator::{PlatformEmulator, PlatformError};
+use crate::arch::emulator::PlatformEmulator;
 
```

Comment:
nit: Could you remove this whitespace line - whitespace lines means cargo fmt will not reorder the use statements correctly. If there are others in this PR please remove those too. Thanks!

Reply:
Done